framework/db/Command.php

```diff
@@ -264,6 +267,9 @@ public function prepare($forRead = null)
             $message = $e->getMessage() . "\nFailed to prepare SQL: $sql";
             $errorInfo = $e instanceof \PDOException ? $e->errorInfo : null;
             throw new Exception($message, $errorInfo, (int) $e->getCode(), $e);
+        } catch (\Throwable $e) {
+            $message = $e->getMessage() . "\nFailed to prepare SQL: $sql";
+            throw new Exception($message, null, (int) $e->getCode(), $e);
```

Comment:
why is `$errorInfo` null?

Reply:
Because `\Throwable` there is not an instance of `\PDOException`. Thus, nowhere to get `errorInfo` from.